manifests/charts/aeraki/templates/configmap.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright Aeraki Authors
```

Comment:
We don't need this configMap in the control plane installation. It's just for initial sidecar configuration.

Reply:
Gotcha